golangci-tool is crashing in CI

golangci-tool is Currently crashing the CI :

```
goroutine 3830 [running]:
github.com/golangci/golangci-lint/vendor/github.com/golangci/go-tools/lint.(*fnVisitor).Visit(0xc4467bba40, 0x11bb740, 0xc435d98630, 0x1, 0xc4212b2b48)
/home/test/go/src/github.com/golangci/golangci-lint/vendor/github.com/golangci/go-tools/lint/lint.go:837 +0x405
github.com/golangci/golangci-lint/vendor/github.com/golangci/go-tools/lint.(*globalVisitor).Visit(0xc4467bba20, 0x11bb740, 0xc435d98630, 0x11b68c0, 0xc4467bba20)
/home/test/go/src/github.com/golangci/golangci-lint/vendor/github.com/golangci/go-tools/lint/lint.go:820 +0xdc
go/ast.Walk(0x11b68c0, 0xc4467bba20, 0x11bb740, 0xc435d98630)
/usr/lib/golang/src/go/ast/walk.go:52 +0x66
go/ast.walkDeclList(0x11b68c0, 0xc4467bba20, 0xc42ca459c0, 0x3, 0x4)
/usr/lib/golang/src/go/ast/walk.go:38 +0x81
go/ast.Walk(0x11b68c0, 0xc4467bba20, 0x11bb6c0, 0xc4367f9300)
/usr/lib/golang/src/go/ast/walk.go:353 +0x2672
github.com/golangci/golangci-lint/vendor/github.com/golangci/go-tools/lint.NodeFns.func1(0xc44555f360, 0xc446846780, 0xc4466bd9f0)
/home/test/go/src/github.com/golangci/golangci-lint/vendor/github.com/golangci/go-tools/lint/lint.go:787 +0x96
created by github.com/golangci/golangci-lint/vendor/github.com/golangci/go-tools/lint.NodeFns
/home/test/go/src/github.com/golangci/golangci-lint/vendor/github.com/golangci/go-tools/lint/lint.go:784 +0x112
make: *** [test] Error 2
```

GolangCI issue: https://github.com/golangci/golangci-lint/issues/260#issuecomment-433850010

--- a/.golangci.yaml
+++ b/.golangci.yaml
@@ -9,6 +9,7 @@ linters:
     - lll
     - gosec
     - prealloc
+    - megacheck
   enable-all: true
 
 issues:

